dotnet/src/SemanticKernel/AI/ImageGeneration/IImageGeneration.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.AI.ImageGeneration;
+
+public interface IImageGeneration
+{
+    /// <summary>
+    /// Generate an image matching the given description
+    /// </summary>
+    /// <param name="description">Image description</param>
+    /// <param name="width">Image width in pixels</param>
+    /// <param name="height">Image height in pixels</param>
+    /// <param name="cancellationToken">Cancellation token</param>
+    /// <returns></returns>
```

Comment:
Would be nice to describe here what it returns. It could be image URL or base64 image I guess.